str_analysis/data/known_pathogenic_variant_catalog__GRCh37__58_loci_without_offtargets.json

```diff
@@ -0,0 +1,1405 @@
+            "OMIM": "117210",
+            "PathogenicMin": 500
+         }
+      ]
+   },
+   {
+      "LocusId": "C9ORF72",
+      "LocusStructure": "(GGCCCC)*",
+      "ReferenceRegion": "chr9:27573526-27573544",
+      "VariantType": "RareRepeat",
+      "InExpansionHunterVariantCatalog": true,
+      "Gene": "C9orf72",
+      "Inheritance": "AD",
+      "GeneRegion": "intron",
+      "DiscoveryMethod": "L, WGS",
+      "DiscoveryYear": 2011,
+      "Diseases": [
+         {
+            "Symbol": "FTDALS1",
+            "Name": "Amyotrophic lateral sclerosis and/or frontotemporal dementia",
+            "Inheritance": "AD",
+            "OMIM": "105550",
+            "NormalMax": 23,
+            "PathogenicMin": 31,
+            "IntermediateRange": "20-30"
+         }
+      ]
+   },
+   {
+      "LocusId": "CACNA1A",
+      "LocusStructure": "(CTG)*",
+      "ReferenceRegion": "chr19:13318672-13318711",
+      "VariantType": "Repeat",
+      "InExpansionHunterVariantCatalog": true,
+      "Gene": "CACNA1A",
+      "Inheritance": "AD",
+      "GeneRegion": "exon - polyglutamine",
+      "DiscoveryMethod": "L, ExScr",
+      "DiscoveryYear": 1997,
+      "Diseases": [
+         {
+            "Symbol": "SCA6",
+            "Name": "Spinocerebellar ataxia 6",
+            "Inheritance": "AD",
+            "OMIM": "183086",
+            "NormalMax": 18,
+            "PathogenicMin": 21,
+            "IntermediateRange": "19-19"
+         }
+      ]
+   },
+   {
+      "LocusId": "CNBP",
+      "LocusStructure": "(CAGG)*(CAGA)*(CA)*",
+      "ReferenceRegion": [
+         "chr3:128891419-128891499",
+         "chr3:128891499-128891539",
+         "chr3:128891539-128891575"
+      ],
+      "VariantId": [
+         "CNBP",
+         "CNBP_CAGA",
+         "CNBP_CA"
+      ],
+      "VariantType": [
+         "Repeat",
+         "Repeat",
+         "Repeat"
+      ],
+      "InExpansionHunterVariantCatalog": true,
+      "Gene": "CNBP",
+      "Inheritance": "AD",
+      "GeneRegion": "intron",
+      "DiscoveryMethod": "L, ExScr",
+      "DiscoveryYear": 2001,
+      "Diseases": [
+         {
+            "Symbol": "DM2",
+            "Name": "Myotonic dystrophy 2",
+            "Inheritance": "AD",
+            "OMIM": "602668",
+            "NormalMax": 29,
+            "PathogenicMin": 55,
+            "IntermediateRange": "30-55"
+         }
+      ]
+   },
+   {
+      "LocusId": "COMP",
+      "LocusStructure": "(GTC)*",
+      "VariantType": "Repeat",
+      "ReferenceRegion": "19:18896844-18896859",
+      "RepeatUnit": "GTC",
+      "InExpansionHunterVariantCatalog": false,
+      "Gene": "COMP",
+      "Inheritance": "AD",
+      "GeneRegion": "exon - poly-aspartic-acid",
+      "Diseases": [
+         {
+            "Symbol": "EDM1",
+            "Name": "Multiple epiphyseal dysplasia",
+            "Inheritance": "AD",
+            "OMIM": "132400",
+            "NormalMax": 5,
+            "PathogenicMin": 6
```

Comment:
I thought that repeats below 5 for COMP were also pathogenic (from the comment in `check_combined_results_tsv_for_pathogenic_repeats.py`)?

Reply:
yes!!